README.md

```diff
@@ -153,7 +153,54 @@ Example output:
 
 # Usage in CI Systems
 
-> 🚧 COMING SOON 🚧
+> 🚧 COMING SOON: native docker image & GitHub action 🚧
+
+## GitHub Actions workflows
+
+Here is an example of a simaple GitHub Actions workflow that uses `kubectl-validate` to validate Kubernetes manifests.
+This workflow will run on every pull request to the `main` branch and will fail if any of the manifests in the dir `k8s-manifest/` are invalid.
+
+```yaml
+name: kubectl-validate
+on:
+  pull_request:
+    branches:
+      - main
+  
+jobs:
+  k8sManifestsValidation:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo content
+        uses: actions/checkout@v3
+        
+      - name: Setup go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.20'
```

Comment:
I think this is inferred from `go.mod` if missing, we could get rid of this. just FYI.